app.py

```diff
@@ -59,9 +67,10 @@ def str2bool(value: str) -> bool:
     stack_name=f"geospatial-data-lake-api-{ENV}",
     env={"region": os.environ["CDK_DEFAULT_REGION"], "account": os.environ["CDK_DEFAULT_ACCOUNT"]},
     datasets_table=storage.datasets_table,
+    datalake_users_role=users.datalake_users_role,
 )
 
-api = StagingStack(
+staging = StagingStack(
```

Comment:
This variable is unused, so we should just remove it.

Reply:
It is just fixing typo, not relevant to scope of this change and will be fixed by other your PRs later. Will drop this change.

Reply:
Change dropped in 6a1530df5f99fdf5474b256ba731f77417cd6119